packages/siaviewnode-client/src/components/Dashboard.tsx

```diff
@@ -0,0 +1,161 @@
+                                    {name: 'Hour', "Spent": stats.money_spent["1"]},
+                                    {name: 'Day', "Spent": stats.money_spent["24"]},
+                                    {name: 'Week', "Spent": stats.money_spent["168"]},
+                                    {name: 'Month', "Spent": stats.money_spent["720"]},
+                                    {name: 'Quarter', "Spent": stats.money_spent["2160"]},
+                                ]} series={{"Spent": theme.colors.primary,}}
+                            />
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col xs={12} md={6}>
+                            <StatBarChart title="Download throughput" units="Mbps" icon={faDownload}
+                                data={[
+                                    {name: 'Last Hour', p80: stats.download_throughput["p80-1"], p95: stats.download_throughput["p95-1"], p99: stats.download_throughput["p99-1"]},
+                                    {name: 'Last Day', p80: stats.download_throughput["p80-24"], p95: stats.download_throughput["p95-24"], p99: stats.download_throughput["p99-24"]},
+                                    {name: 'Last Week', p80: stats.download_throughput["p80-168"], p95: stats.download_throughput["p95-168"], p99: stats.download_throughput["p99-168"]},
+                                ]} series={{"p80": colorChanger(theme.colors.primary, 50), "p95": theme.colors.primary, "p99": colorChanger(theme.colors.primary, -50)}}
+                            />
+                        </Col>
+                        <Col xs={12} md={6}>
+                            <StatBarChart title="Upload throughput" units="Mbps" icon={faUpload}
+                                data={[
+                                    {name: 'Last Hour', p80: stats.upload_throughput["p80-1"], p95: stats.upload_throughput["p95-1"], p99: stats.upload_throughput["p99-1"]},
+                                    {name: 'Last Day', p80: stats.upload_throughput["p80-24"], p95: stats.upload_throughput["p95-24"], p99: stats.upload_throughput["p99-24"]},
+                                    {name: 'Last Week', p80: stats.upload_throughput["p80-168"], p95: stats.upload_throughput["p95-168"], p99: stats.upload_throughput["p99-168"]},
+                                ]} series={{"p80": colorChanger(theme.colors.primary, 50), "p95": theme.colors.primary, "p99": colorChanger(theme.colors.primary, -50)}}
+                            />
+                        </Col>
+                    </Row>
+                </Container>
+            </Row>
+        </Container>
+    </div> )
+}
+
+
+// Custom component: StatBox
+function StatBox(props) {
+    return <Col md={5} lg={3} style={{border: '2px solid ' + theme.colors.statsElementsBorder, borderRadius: "15px", paddingTop: "20px", paddingBottom: "20px", justifyContent: 'center', textAlign: 'center', marginBottom: 50}}>
```

Comment:
Quite a lot of inline styling going on here, perhaps it makes sense to add styled components (https://styled-components.com/docs/basics). Definitely not blocking this PR or anything like that though, just sharing in case you're not familiar with styled components